packages/mobile/src/account/Persona.tsx

```diff
@@ -56,12 +58,13 @@ const Persona = ({ kycStatus, text }: Props) => {
           TAG,
           `Inquiry completed for ${inquiryId} with attributes: ${JSON.stringify(attributes)}`
         )
-        // TODO [Lisa]: Add event handling for KYC approval when Persona component is integrated
```

Comment:
It would have been nice to start the 'verifying  your identity' view with the spinner right here in the onSuccess hook. Alas, the Persona modal closes *before* calling this code so the user experiences a brief flash when I tried to do it that way.